internal/acceptance/openstack/objectstorage/v1/objects_test.go

```diff
@@ -68,7 +70,6 @@ func TestObjects(t *testing.T) {
 
 	// List all created objects
 	listOpts := objects.ListOpts{
-		Full:   true,
```

Comment:
List option `Full: false` (`accept: text/plain`) doesn't work and should not be used